tests/unit/builders/indexable-post-builder-test.php

```diff
@@ -300,6 +300,7 @@ public function test_build() {
 		$this->indexable->orm->expects( 'set' )
 			->with( 'open_graph_image', 'http://basic.wordpress.test/wp-content/uploads/2020/07/WordPress5.jpg' );
 		$this->indexable->orm->expects( 'set' )
+			// phpcs:ignore Yoast.Yoast.AlternativeFunctions.json_encode_json_encodeWithAdditionalParams
```

Comment:
@jrfnl Shouldn't this have a reason at the end of the line?

Reply:
Same for all ignores below

Reply:
*grin* you caught me ;-) Yes, it should, but I figured it was obvious as this is test code and we're mocking WP by using PHP native functions ;-)

I'll get that fixed up.